packages/block-editor/src/components/block-lock/toolbar.js

```diff
@@ -33,10 +34,23 @@ export default function BlockLockToolbar( { clientId, wrapperRef } ) {
 	useEffect( () => {
 		if ( isFirstRender.current ) {
 			isFirstRender.current = false;
+			hasModalOpened.current = false;
```

Comment:
Since `hasModalOpened.current` is already initialized as `false` , is this line necessary?